mlir/test/Dialect/Vector/linearize.mlir

```diff
@@ -306,3 +306,14 @@ func.func @test_vector_insert_scalable(%arg0: vector<2x8x[4]xf32>, %arg1: vector
   // ALL: return %[[RES]] : vector<2x8x[4]xf32>
   return %0 : vector<2x8x[4]xf32>
 }
+
+// -----
+
+// ALL-LABEL: test_vector_extract_scalar
+func.func @test_vector_extract_scalar() {
+  %cst = arith.constant dense<[1, 2, 3, 4]> : vector<4xi32>
+  // ALL:     vector.extract
+  // ALL-NOT: vector.shuffle
```

Comment:
this would still pass if a shuffle came before `vector.extract`, have to check both sides with CHECK-NOT 
```suggestion
  // ALL-NOT: vector.shuffle
  // ALL:     vector.extract
  // ALL-NOT: vector.shuffle
```

Reply:
Done